src/pages/scheduler/components/SchedulerCalendar.tsx

```diff
@@ -149,12 +150,26 @@ export const SchedulerCalendar = ({ term, courseCalendarEvents = [] }: Scheduler
         // if event does not have a scheduled time, move on.
         if (calendarEvent.meetingTime.time.indexOf('TBA') !== -1) return;
 
+        if (
+          calendarEvent.meetingTime.time.split(`- `)[1].split(` `)[1] === `pm` &&
+          parseInt(calendarEvent.meetingTime.time.split(`- `)[1].split(':')[0]) !== 12
+        ) {
+          let theTime = parseInt(calendarEvent.meetingTime.time.split(`- `)[1].split(`:`)[0]) + 12;
+          if (theTime > latestHour && theTime < 24) {
+            setLatestHour(theTime + 1);
```

Comment:
`theTime` is a bit sus as a variable name. Also it can be `const`.

Reply:
@JoshoTheMosho This doesn't need to be here anymore, lines 183-187 replaced this code 😃